### ISSUE TITLE
Connecting more than one Monitor causes Idle Power usage to increase

### ISSUE THREAD

**Issue by shenef** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#164]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows

### Processor / Processor Number [Required]

Intel i5-12600k

### Graphic Card [Required]

Arc A770 16GB

### GPU Driver Version [Required]

- [ ] 31.0.101.3491 (*latest*)
- [X] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [ ] Windows 11 22H2
- [ ] Windows 11 21H1 (*October 2021 Update*)
- [ ] Windows 10 21H2 (*November 2021 Update*)
- [ ] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Other (Please specify)

### Other Windows build number

Windows 11 22H2 (25227)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/9873766/igcit_ssu.txt)

### Description and steps to reproduce [Required]

When only one Screen is connected to the GPU the Idle power draw is ~5-7W. ([recommended configuration](https://www.intel.com/content/www/us/en/support/articles/000092564/graphics.html) already applied)   
When a second or third Screen is connected, the idle power draw rises to 35-40W.
Tested with a 1600x900@60Hz Monitor and a 1920x1080@60Hz TV.

### Crash dumps [Required]

[igcit-dumps_5c87d9bc.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/9873811/igcit-dumps_5c87d9bc.zip)

**Comment by IntelSupport-Arun** [comment]
Hi @shenef ,

For further investigation we would like to know below information from your side:

- Make and model the display used to see the issue?
- Single display,clone, or extended ?
- AC or DC mode,i.e. is the power cable plugged in or not?
- We would appreciate it if you could provide a screenshot or short video of the issue in order to better understand.

**Comment by bizude** [comment]
It's normal for idle power consumption to increase when using other vendor's GPUs, I don't know if this is something that can be "fixed" per se

**Comment by IntelSupport-Arun** [comment]
Hi @bizude ,

Could you provide the details which we requested earlier?

**Comment by shenef** [comment]
@IntelSupport-Arun sorry for the late reply.

> Make and model the display used to see the issue?

- LG M237WDP
- LG 29UM58
- HP 2009m
- another monitor I currently don't have access to

The problem persists as long as more than one screen is connected and the type of connection (HDMI or DP) has no impact.

---

> Single display,clone, or extended ?

The Problem persists across all three display modes.

---

> AC or DC mode,i.e. is the power cable plugged in or not?

I'm not certain what exactly you are asking so i'll give as much information as possible
- The card is, in addition to the PCI-e slot, powered by one 8-pin and one 6-pin connector.
- As long as more than one display cable is plugged into the card, the power draw increases. The power draw does not decrease if additional screens are powered down or even fully disconnected from the power grid.

---

> We would appreciate it if you could provide a screenshot or short video of the issue in order to better understand.

- Three screens connected ![image](https://user-images.githubusercontent.com/20268890/200413201-7a2083db-b2bf-4724-9aae-661f877035c7.png)
- Two screens connected ![image](https://user-images.githubusercontent.com/20268890/200413375-01ddb266-064c-4c6d-a1cb-a37790cb908b.png)
- One screen connected ![image](https://user-images.githubusercontent.com/20268890/200414933-c2c3bc86-bdf2-4921-91c4-cb9fea715487.png)

Additionally i want to mention that the card misreporting can pretty much be ruled out since it's noticeably louder and produces noticeably more heat

**Comment by IntelSupport-Arun** [comment]
Hi @shenef ,

Thank you for the detailed information.
I have a quick check with you, may I know the Motherboard make and BIOS details?

**Comment by IntelSupport-Rozilah** [comment]
Thanks @shenef for the confirmation.

**Comment by IntelSupport-Rozilah** [comment]
hi @shenef,
The case has been verified on our side and we are able to reproduce it with our configurations as below:
Processor: i7-12700K
Graphic Driver version:  31.0.101.3490 and  31.0.101.3793
We will be working on a fix. I want to give you a heads-up that the fix may take 3 to 6 months to be included.

**Comment by IntelSupport-Rozilah** [comment]
hi @shenef,
just for confirmation, did you configure both BIOS and Windows* Power settings on your system from [here](https://www.intel.com/content/www/us/en/support/articles/000092564/graphics.html), if yes does the idle power is lower?

**Comment by shenef** [comment]
@IntelSupport-Rozilah Yes, as mentioned in my initial report, i have applied those recommended power settings previous to my testing and i can confirm it reduces idle power when a single monitor is connected but the idle power is still high if multiple monitors are connected.

**Comment by IntelSupport-Rozilah** [comment]
hi @shenef and @jschoch
could you guys try using the same resolution on all the monitors connected to the Arc?, please let me know your observation.
Thanks!.

**Comment by shenef** [comment]
I've changed the resolution on all screens to 1280x720px.
This had no impact on the idle power, it stayed at 38-40W. 
Rebooting the machine did not result in any changes
I'm already on Driver .3802

**Comment by IntelSupport-Rozilah** [comment]
hi @shenef , 
Thanks for the update, i have updated your observation to our internal and stay tuned for the fix !.

**Comment by IntelSupport-Rozilah** [comment]
Hi all,
There is an update from internal, a fix for this issue is now been developed. However, it will take 3 to 6 months before the fix is merged into a public driver. Stay Tuned!

**Comment by ghchris2021** [comment]
@IntelSupport-Rozilah 

Hi as a related question, I see that you've developed a candidate fix for this issue but I haven't seen any more in depth details as to what is possible and expected to be affected resulting from the problem analysis / mitigation strategy which I am curious about .

May I please be informed based on the current estimation of the design / problem analysis / mitigation strategy:

1: Is the expected outcome that the card will draw the lower mentioned power range (circa 5-15W vs. 30-45W general area) regardless of the number of connected monitors when the GPU activity required can be fairly minimal e.g. just showing a static minimally graphically intensive desktop on 1, 2, 3, 4 displays?

2: Is the expected outcome that the GPU power consumption will be in the lower of the listed ranges when any attached monitors are in sleep / standby / unpowered or similar modes since it was mentioned that even in these cases the GPU power remained high i.e. does the change relate to improving power use in "active" and "idle" and "standby" etc. use case modes?

3: It was mentioned that this issue was confirmed to be present for Windows 11 OSs but being a power management issue that could affect multiple OSs, does the same problem affect supported and nominally configured LINUX OS use?  Is the proposed mitigation code / configuration change also expected to be useful for and commensurately made available to improve single or multi-monitor attached power use for e.g. A770 tier GPUs under LINUX due to the similar use cases / implementations which caused this issue for W11?

4: If it is the case that there may be a corresponding fix / improvement in the GPU power management for LINUX OS environments, is the same 3-6 month time frame applicable before that update would be made visible externally as a proposed / testing update to the open LINUX code base which would subsequently become part of a particular stable release?

5: For the Intel A770 LE (for example) when the card is able to operate in the lower listed range (e.g. 5-15W or so) due to basically being idle with some monitors attached will the GPU fans be able to remain totally stopped at 0RPM assuming the actual system / motherboard / GPU sensed temperatures remain low and near ambient and there is good general case ventilation for both LINUX and Windows OS environments, or how is it expected to be able to function wrt. GPU FAN speed / duty cycle reduction in the low power "ON" / "IDLE" cases?  I am interested to know under what conditions it is or might be possible to have the GPU FANs stay off @ 0 RPM under the various supported OSs.

 
> Hi all, There is an update from internal, a fix for this issue is now been developed. However, it will take 3 to 6 months before the fix is merged into a public driver. Stay Tuned!

**Comment by EstebanIntel** [comment]
Hi all,

Just wanted to update you on the status of this issue.

I can't share specifics on the expected power when idle yet. But I do want to set the right expectations: the support for the is deeply tied to the number of displays been connected and the resolutions of said displays. So, from our initial tests, with this internal build we are able to go into the low power state when we have 2x1080p@60 displays and 1x4k@60 display. For 3x or more displays and for single displays using 5k@60 or greater resolution, the Arc cards will still have a higher power draw when idle.

**Comment by clivem** [comment]
So the expectation for someone using 2 displays at 1080p/1440p @ 144/165 Hz..... The low power state/mode should be achievable? 
I would guess at a something like a 20W reduction on the 40W idle, from just down-clocking the memory clock from the current 2187MHz at idle. Is this where the power saving will come from, down-clocking the memory?

**Comment by EstebanIntel** [comment]
@clivem, unfortunately, I cannot go into technical details on how this will be achieved. But I want to clearly set the right expectations of which combinations of number of displays and resolutions will allow Arc GPUs to enter the low power state when idle once this fix is merged into the driver:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/esteban2/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/esteban2/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



# of   Displays | Can Arc   A-Series enter the low power state when idle?
-- | --
1 | yes, up to 4k@60
2 | yes, up to 1080p@60
3 | no
4 | no



</body>

</html>

**Comment by EstebanIntel** [comment]
@clivem, unfortunately, I cannot go into technical details on how this will be achieved. But I want to clearly set the right expectations of which combinations of number of displays and resolutions will allow Arc GPUs to enter the low power state when idle once this fix is merged into the driver:

| # of Displays  | Can Arc A-Series enter the low power state when idle? |
| ------------- | ------------- |
| 1  | yes, up to 4k@60 |
| 2  | yes, up to 1080p@60 |
| 3  | no |
| 4  | no |

**Comment by EstebanIntel** [comment]
Hi @clivem, we can ensure it will, but the latest driver version [31.0.101.4146](https://www.intel.com/content/www/us/en/download/726609/intel-arc-iris-xe-graphics-whql-windows.html?) has the fix for this issue, so you can give a test. Also, please be sure to follow the recommended configuration described here: [https://www.intel.com/content/www/us/en/support/articles/000092564/graphics.html](https://www.intel.com/content/www/us/en/support/articles/000092564/graphics.html)

@shenef can you please test it as well?

**Comment by shenef** [comment]
@EstebanIntel Seems to be a success! Idle Power with more than one monitor is down to normal levels and also seems a bit lower in general.

4091:
Single 2560x1080: 11 W
2560x1080 + 1600x900: 38-40 W

4146:
Single 2560x1080: 7-8 W
2560x1080 + 1600x900: 8-9 W

**Comment by EstebanIntel** [comment]
Great news!!

@IGCIT this seems resolved now, can the issue be closed?

**Comment by EmptyKeys** [comment]
This still does not work for me in dual monitor setup. GPU shows 40W usage.

**Comment by shenef** [comment]
@EmptyKeys What resolutions and refresh rates are your monitors running at?  
You can check this message for the scenarios Arc is expected to go into the low power state: https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/issues/164#issuecomment-1381085325

**Comment by EmptyKeys** [comment]
Dual setup
1920x1080 @60
2560x1440 @155

**Comment by EmptyKeys** [comment]
I see, that's unfortunate.

**Comment by jasonpel** [comment]
Hello,

Just installed a Sparkle ARC A750 ORC OC Edition GPU. I have three monitors (1920x1080@60HZ) plugged into the display ports. I have the latest GPU driver installed along with Intel Arc Control.

My GPU is pulling 43W at idle, similar to those who posted above. Could this be contributing to the high GPU temperature I am seeing at idle?

And was there ever a fix to lower the GPU Power Draw when using three monitors?

Thanks!

![Screenshot 2023-11-25 215857](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/27898494/707a1007-7fb8-4ba9-8478-893d7c46cea2)
![Screenshot 2023-11-25 220030](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/27898494/2d7c670a-95cf-4175-bcc7-496b9e5d05d1)

**Comment by EmptyKeys** [comment]
No point on beating dead horse. Topic is already closed.

Intel GPUs are simply not finished products and I don't expect that this will be ever fixed. If you are not OK with that just buy AMD or Nvida.
